apis/v1alpha1/generator.yaml

```diff
@@ -9,15 +9,8 @@ resources:
         404:
           code: ClusterNotFoundFault
       terminal_codes:
-        - ClusterAlreadyExistsFault
-        - ClusterQuotaForCustomerExceededFault
-        - NodeQuotaForClusterExceededFault
-        - NodeQuotaForCustomerExceededFault
-        - InsufficientClusterCapacityFault
-        - ShardsPerClusterQuotaExceededFault
         - InvalidParameterValueException
         - InvalidParameterCombinationException
-        - TagQuotaPerResourceExceeded
         - NoOperationFault
     renames:
       operations:
```

Comment:
Please include the `ack-generate-metadata.yaml` file so we know what ack-generate version you used

Reply:
Added